repos.d/rpm/openmandriva.yaml

```diff
@@ -0,0 +1,42 @@
+###########################################################################
+# OpenMandriva
+###########################################################################
+- name: openmandriva_3
+  type: repository
+  desc: OpenMandriva 3.x
+  family: openmandriva
+  color: '045e9d'
+  sources:
+      # XXX: */backports_testing and */updates_testing not included, are these needed?
```

Comment:
There are no `*/backports_testing and */updates_testing`, please remove comment.